extensions/jackson/runtime/src/main/java/io/quarkus/jackson/runtime/ObjectMapperProducer.java

```diff
@@ -28,6 +29,10 @@ public ObjectMapper objectMapper(Instance<ObjectMapperCustomizer> customizers,
             // this feature is enabled by default, so we disable it
             objectMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
         }
+        if (!jacksonConfigSupport.isWriteDatesAsTimestamps()) {
```

Comment:
Since the default value is `false` this is now the default behavior, right?

Reply:
I'm talking about the default behavior of Jackson.

Reply:
My point is that your `if` statement will always be executed since this is a new property

Reply:
But users can disable it, right?

I don't see why what Guillaume has done would be a problem

Reply:
No problem at all, just something that needs to be in the migration guide in case someone complains that their JSON serialized dates changed 😉

Reply:
Yeah, for sure. This **is** a breaking change and both Guillaume and I mentioned that in the dev mailing list thread :)

Reply:
I had already documented it in the Migration guide for 1.13: https://github.com/quarkusio/quarkus/wiki/Migration-Guide-1.13 .